install/installer/pkg/config/v1/config.go

```diff
@@ -227,6 +228,10 @@ type DatabaseCloudSQL struct {
 	Instance       string    `json:"instance" validate:"required"`
 }
 
+type SslOptions struct {
+	CustomCA string
```

Comment:
```suggestion
	CustomCA ObjectRef
```

All secrets are referred to as `ObjectRef` to point out that this is a reference to a secret, not the secret itself.